src/sdk-reference/go/1/getting-started/index.md

```diff
@@ -0,0 +1,117 @@
+
+[snippet=init]
+
+This program will initialize our Kuzzle creating an index and a collection
+we'll use in the next section.
+Run the program with the following command:
+
+```bash
+$ go run init.go
+Connected!
+Index nyc-open-data created!
+Collection yellow-taxi created!
+```
+
+Congratulations, you performed a first connection to Kuzzle with a Go program.
+You are now able to:
+* Load the `Kuzzle Go SDK` from your `GOPATH`.
+* Instantiate a protocol (here `websocket`) and a Kuzzle SDK instance.
+* Connect to Kuzzle instance running on `localhost` with the `websocket` protocol.
+* Create the `nyc-open-data` index.
+* Create the `yellow-taxi` collection (within the `nyc-open-data` index).
+
+## Create your first document
+
+Now you successfully connected to your Kuzzle server with the Go SDK and have an index and a collection, it's time to manipulate data. Kuzzle uses `indexes` that contain `collections`.
+Those `collections` store `documents`.
+Create a `document.go` file in the playground and add this code:
+
+[snippet=document]
+
+As you did before, run your program:
+
+```bash
+$ go run document.go
+Connected!
+New document added to yellow-taxi collection!
+```
+
+You can perform other actions such as [delete]({{ site_base_path }}sdk-reference/go/1/document/delete),
+[replace]({{ site_base_path }}sdk-reference/go/1/document/replace) or [search]({{ site_base_path }}sdk-reference/go/1/document/search) documents. There are also other way to manipulate data in Kuzzle
+like our [Admin Console]({{ site_base_path }}guide/2/essentials/installing-console/), the [Kuzzle HTTP API]({{ site_base_path }}api/1/essentials/connecting-to-kuzzle/) or your [own protocol](protocols/1/essentials/getting-started/).
+
+Now you know how to:
+* Store data in your Kuzzle server with documents and access it.
+
+## Subscribe to realtime document notifications (pub/sub)
+
+Time to use realtime with Kuzzle. Create a new file `realtime.go` containing:
+
+[snippet=realtime]
+
+This program subscribe to `yellow-taxi` collection. For each new document filter matching, we'll receive notification.
+Run your program:
+
+```bash
+$ go run realtime.go
+Connected!
+Successfully subscribing!
+New document added to yellow-taxi collection!
+Driver John born on 1995-11-27 got a B license.
+```
+
+Now, you know how to:
+* Create realtime filters.
+* Subscribe to notifications.
+
+<div class="alert alert-info">
+If you want to go deeper in the SDK, source code is available on <a href="https://github.com/kuzzleio/sdk-go">Github</a>.
+</div>
+
+<div class="alert alert-info">
+Having trouble? Get in touch with us on <a href="https://gitter.im/kuzzleio/kuzzle">Gitter!</a> We're happy to help.
+</div>
+
+## Where do we go from here?
+
+Now that you're more familiar with Kuzzle Go SDK, dive even deeper to learn how to leverage its full capabilities:
+
+* take a look at the <a href="{{ site_base_path }}sdk-reference/js/6">SDK Reference</a>
```

Comment:
This link actually refers to... the Javascript Getting Started page. Fixing the link to make it point to the Go SDK directory won't be much better, as this will make it point to the current page.

```suggestion
* discover what this SDK has to offer by browsing other sections of this documentation
```